### PR TITLE
Port relay_field from ROS 1

### LINKED ISSUE #40: Port relay_field from ROS1
## Description

Would be nice to have https://github.com/ros/ros_comm/pull/639 in ROS2

### PR DESCRIPTION
**Pull request by esteve**
Closes #40

### CODE REVIEW

**Comment by esteve** [comment]
This PR does not support the `now` and `auto` keywords like `relay_field` supported in ROS 1. This functionality is in `rosdil_runtime_py` now, but it hasn't been released to Humble yet (it's merged https://github.com/ros2/rosidl_runtime_py/pull/20, but there hasn't been a new release of `rosidl_runtime_py` for Humble).

I can update this PR once there's a new release of `rosidl_runtime_py`, but that may break Galactic and Foxy, as it's not been ported there yet.

**Review by emersonknapp — APPROVED** [review]
LGTM

**Comment by esteve** [comment]
@emersonknapp I've fixed the flake8 errors, could you trigger the CI again? Thanks.

**Comment by esteve** [comment]
@emersonknapp thanks for fixing the rest of the flake8 errors, I've fixed the DCO error by amending your last commit with my signature, hope that's ok!

File: README.md [hunk 1]
```diff
@@ -37,9 +37,27 @@ ros2 run topic_tools relay base_scan my_base_scan
...
     - If True, only subscribe to `input_topic` if there is at least one subscriber on the `output_topic`
 
+### RelayField
+
+RelayField is a ROS 2 node that allows to republish data in a different message type
+
+#### Usage
+
+```
+ros2 run topic_tools transform <input topic> <output topic> <output type> [<expression on m>] [--import <modules>] [--field <topic_field>]
```
Comment (ymd-stella) [inline README.md:47]: Something is wrong because transform is being used incorrectly.

Comment (christophebedard) [reply README.md:47]: can you open a new issue (or a PR)?

Comment (ymd-stella) [reply README.md:47]: Done. #77

Comment (esteve) [reply README.md:47]: Oops, you're right. The changes in #77 look good to me. Thanks!

**Pull request merged**